crates/sui-config/src/genesis.rs

```diff
@@ -352,6 +352,10 @@ pub struct GenesisChainParameters {
     ///   kicked from the validator set
     #[serde(default)]
     pub governance_start_epoch: u64,
+
+    /// The duration of an epoch, in milliseconds.
+    #[serde(default = "GenesisChainParameters::test_epoch_duration_ms")]
```

Comment:
Why is default test_epoch_duration_ms instead of prod value?

Reply:
Having the default as the test value does make testing potentially a bit easier, as the prod value will need to be explicitly audited during the genesis ceremony.